From jaswinder@kernel.org  Thu Aug 13 16:57:42 2009
From: Jaswinder Singh Rajput <jaswinder@kernel.org>
Date: Thu, 13 Aug 2009 13:11:12 +0530
Subject: Staging: rt2860: includecheck fix: rt_linux.h
To: Greg KH <greg@kroah.com>, Greg Kroah-Hartman <gregkh@suse.de>, devel@driverdev.osuosl.org
Message-ID: <1250149272.6116.7.camel@ht.satnam>



fix the following 'make includecheck' warning:

  drivers/staging/rt2860/rt_linux.h: linux/wireless.h is included more than once.

Signed-off-by: Jaswinder Singh Rajput <jaswinderrajput@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2860/rt_linux.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/staging/rt2860/rt_linux.h
+++ b/drivers/staging/rt2860/rt_linux.h
@@ -63,10 +63,6 @@
 #include <linux/ctype.h>
 #include <linux/vmalloc.h>
 
-
-#ifdef RT30xx
-#include <linux/wireless.h>
-#endif
 #include <net/iw_handler.h>
 
 // load firmware
